specification.md

```diff
@@ -1579,6 +1584,57 @@ immediately after successful connection each Agent will acquire their own unique
 connection credentials. This way individual Agent's credentials may be revoked
 without disrupting the access to all other Agents.
 
+#### Agent-initiated CA trust Flow
+
+Status: [Development]
+
+This is an Agent-initiated flow that allows an agent to connect to a Server without
```

Comment:
Right now this does not use an explicit signal (from the agent to the server).

We can add one as a part of the `OpAMPConnectionSettingsRequest` message.
But then I'm concerned that we don't have a way for the agent to signal to the server that offered settings have been attempted, but have failed.